vignettes/read-myacc-csv.Rmd

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
The file name is now "vignettes/read-myacc-csv.Rmd" and CRAN will use this as URL.
For the other vignettes I tried to avoid spaces or hyphens to avoid risk of ending up with URL issues.

So, can you please remove the hyphens such that it becomes  "vignettes/readMyAccCsv.Rmd" or "vignettes/readmyacccsv.Rmd" the user will hardly notice this as it is only in the URL.